Scripts/World/Places/hunt_spot.gd

```diff
@@ -0,0 +1,78 @@
+class_name HuntSpot
+extends Place
+
+## Hunt Spot starts with the lvl 1 army present on top of it
+##
+##
+
+## Setup Variables
+var neutral_armies : Array[PresetArmy]
+var material_rewards : Array[Goods]
+var army_respawn_timer : int = 1 # in turns
+
+## local variables
+var _level_manager : int = 0
```

Comment:
rename  to `_current_level`
manager suggests it does something else
backing field should have name similar to property masking it